Makefile

```diff
@@ -205,18 +201,6 @@ server.start: ## Run the local webserver using Symfony
 server.stop: ## Stop the local webserver
 	${SYMFONY} local:server:stop
 
-###
-### GITHUB CODESPACES
-### ¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯
-
-codespace.hostname: ${APP_DIR}/.env.local ## Init the SYLIUS_FIXTURES_HOSTNAME variable in .env.local
-ifdef CODESPACES
-	echo SYLIUS_FIXTURES_HOSTNAME=${CODESPACE_NAME}-8000.${GITHUB_CODESPACES_PORT_FORWARDING_DOMAIN} >> ${APP_DIR}/.env.local
-endif
-
-${APP_DIR}/.env.local:
-	touch ${APP_DIR}/.env.local
-
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** kept these lines or remove the `.devcontainer` folder‏

Reply:
I removed `.devcontainer` for now